specification/dataprotection/resource-manager/Microsoft.DataProtection/preview/2023-04-01-preview/dataprotection.json

```diff
@@ -0,0 +1,7956 @@
+              "$ref": "#/definitions/AzureBackupRecoveryPointResourceList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-odata": "#/definitions/RecoveryPointsFilters",
+        "x-ms-examples": {
+          "List Recovery Points in a Vault": {
+            "$ref": "./examples/BackupInstanceOperations/ListRecoveryPoints.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataProtection/backupVaults/{vaultName}/backupInstances/{backupInstanceName}/recoveryPoints/{recoveryPointId}": {
+      "get": {
+        "tags": [
+          "RecoveryPoint"
+        ],
+        "description": "Gets a Recovery Point using recoveryPointId for a Datasource.",
+        "operationId": "RecoveryPoints_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v4/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v4/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v4/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/VaultName"
+          },
+          {
+            "$ref": "#/parameters/BackupInstanceName"
+          },
+          {
+            "name": "recoveryPointId",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/AzureBackupRecoveryPointResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Recovery Point": {
+            "$ref": "./examples/BackupInstanceOperations/GetRecoveryPoint.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataProtection/locations/{location}/fetchSecondaryRecoveryPoints": {
+      "post": {
+        "tags": [
+          "FetchSecondaryRecoveryPoints"
+        ],
+        "description": "Returns a list of Secondary Recovery Points for a DataSource in a vault, that can be used for Cross Region Restore.",
+        "operationId": "FetchSecondaryRPs",
```

Comment:
operationIds should be in a form of `Noun_Verb`

Reply:
SecondaryRPs_Fetch

Reply:
Done